recipes/aeron/all/conanfile.py

```diff
@@ -0,0 +1,161 @@
+import os
+import shutil
+import re
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AeronConan(ConanFile):
+    name = "aeron"
+    description = "Efficient reliable UDP unicast, UDP multicast, and IPC message transport"
+    topics = ("conan", "aeron", "udp", "messaging", "low-latency")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/real-logic/aeron/wiki"
+    license = "Apache License 2.0"
```

Comment:
```suggestion
    license = "Apache-2.0"
```

https://spdx.org/licenses/Apache-2.0.html

Reply:
Adjusted